src/pages/settings/InitialSettingsPage.js

```diff
@@ -52,6 +54,12 @@ const propTypes = {
         role: PropTypes.string,
     })),
 
+    /** List of policy members */
+    policyMembers: PropTypes.objectOf(PropTypes.shape({
+        /** ID of the policy */
+        policyID: PropTypes.string,
```

Comment:
Huh? I don't think the policy member has policyID as a key.

Reply:
I guess same comment above about seeing what this key looks like, but this is all policy policy members so 
```
{
     POLICY_MEMBER_LIST_{policyID1}:  {},
     POLICY_MEMBER_LIST_{policyID2}: {},
     POLICY_MEMBER_LIST_{policyID3}: {},
}
```

Reply:
Yes, not sure how to get the ID from the key though.